src/webui/webapplication.cpp

```diff
@@ -367,6 +367,8 @@ void WebApplication::action_command_download()
     CHECK_URI(0);
     QString urls = request().posts["urls"];
     QStringList list = urls.split('\n');
+    bool skipChecking = request().posts["skip_checking"].trimmed() == "true";
+    bool addPaused = request().posts["paused"].trimmed() == "true";
```

Comment:
@buinsky 
Seems `.trimmed()` is the only issue remaining.
What do you think, should this PR be merged now, or delayed until `RequestParser::parseContent()` is re-evaluated and maybe properly fixed?